src/matchers.rs

```diff
@@ -85,42 +87,35 @@ pub enum MatcherType {
     Placeholder,
 }
 
+#[derive(Debug)]
+pub struct MatcherFuncArgs<'a> {
```

Comment:
Can this be called MatcherContext or something?

Reply:
you don't like that the struct that is just supposed to be a wrapper for __arguments__ of the __matcher__ __functions__ is named as such?

Reply:
Yes, I would prefer it be called MatcherContext, as it's the context being passed to matchers that they match with

MatcherFuncArgs just reads really weird

Reply:
I thing `MatcherFuncArgs` reads normal and makes more sense but I don't actually care too much, I don't think `MatcherContext` is destructive